radicle-cli/src/commands/delegate.rs

```diff
@@ -0,0 +1,122 @@
+use std::ffi::OsString;
+use std::str::FromStr;
+
+use anyhow::{anyhow, Context as _};
+
+use radicle::identity::Id;
+use radicle_crypto::PublicKey;
+
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
+mod add;
+mod list;
+mod remove;
+
+pub const HELP: Help = Help {
+    name: "delegate",
+    description: "Manage the delegates of an identity",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad delegate add [<id>] --key <public key>
+    rad delegate remove [<id>] --key <public key>
```

Comment:
So I would flip these around since the `id` is implied if we're in a project dir, whereas the key is always required:

`rad delegate (add|remove) <public-key> [--repo <id>]`

Reply:
Ya, that's fair. Originally I also had the `--name` so it wasn't as clear then :) Thanks!